tests/console/yast2_apparmor.pm

```diff
@@ -102,15 +102,13 @@ sub run {
         # try out with second element in the list
         wait_screen_change { send_key 'tab' };
         wait_screen_change { send_key 'down' };
-        save_screenshot;
         send_key(is_pre_15() ? 'alt-t' : 'alt-c');
+        # toggle takes some seconds:
         wait_still_screen(stilltime => 5);
         if (is_pre_15()) {
             send_key 'tab';
-            wait_still_screen(stilltime => 5);
             send_key 'end';
```

Comment:
two subsequent calls to send_key are dangerous when not synchronized. In most cases the best way is to have a specific needle check as intermediate step. This is most reliable and also the fastest:

```perl
send_key 'tab';
assert_screen 'whatever_field_you_want_to_have_selected_by_tab';
send_key 'end';
```

as an alternative in most cases what should work:

```perl
wait_screen_change { send_key 'tab' };
send_key 'end';
```

Reply:
I used the wait_screen { send_key } as it seems the most legible and simple,
I kept it simple :)

thanks